rotkehlchen/premium/premium.py

```diff
@@ -77,6 +77,18 @@ class UserLimits(TypedDict):
     reports_lookup_limit: int
     # Maximum amount of ETH that can be staked (validator balance limit)
     eth_staked_limit: int
+    # feature flags
+    enable_eth_staking_view: bool
+    enable_assets_historical_view: bool
+    enable_statistics_view: bool
```

Comment:
This includes both graphs and premium analytics?

Reply:
the statistics and the stats based on history events (the wrap)